src/web/src/main/java/tupperdate/web/legacy/routing/recipes/Recipes.get.kt

```diff
@@ -84,7 +84,7 @@ private fun Route.all(store: Firestore) = get {
                 .limit(remaining)
                 .get().await()
         }
-        filtered.addAll(retrieved.filter { it["userId"] != uid })
+        filtered.addAll(retrieved.filter { it["userId"] != id })
```

Comment:
Shouldn't it be

```suggestion
        filtered.addAll(retrieved.filter { it["userId"] != id.uid })
```

instead ?